plugins/modules/proxmox.py

```diff
@@ -51,6 +52,41 @@
       - See U(https://pve.proxmox.com/wiki/Linux_Container) for a full description.
       - Should not be used in conjunction with O(storage).
     type: str
+  disk_volume:
+    description:
+      - Specify a hash/dictionary of the C(rootfs) disk.
+      - See U(https://pve.proxmox.com/wiki/Linux_Container#pct_mount_points) for a full description.
+    type: dict
+    suboptions:
+      storage:
+        description:
+          - V(storage) is the storage identifier of the storage to use for the C(rootfs).
+          - Mutually exclusive with V(host_path).
```

Comment:
Never use `V(...)` for options, only use it for values.
```suggestion
          - Mutually exclusive with O(disk_volume.host_path).
```
Same in other places.